libs/designsystem/src/lib/components/button/button.component.scss

```diff
@@ -101,7 +101,7 @@
   // Only apply focus ring if pointer device can hover
   // (effectively desktop/mouse devices):
   @include focus() {
-    border-color: $focus-ring-color;
+    border-color: $focus-ring-color !important;
   }
 
```

Comment:
This change fixes the issue, and also makes the code simpler.
The frontender does not have to think about focus state, when styling the buttons. Focus will just always overrule what border has been set on the button style.